jboss-maven/configure.sh

```diff
@@ -1,12 +1,21 @@
 #!/bin/sh
-# Add default Maven settings with Red Hat/JBoss repositories
+# Configure module
 set -e
 
 SCRIPT_DIR=$(dirname $0)
 ADDED_DIR=${SCRIPT_DIR}/added
 
+chown -R jboss:root $SCRIPT_DIR
+chmod -R g+rwX $SCRIPT_DIR
+
+# Add default Maven settings with Red Hat/JBoss repositories
 mkdir -p $HOME/.m2
 cp -p ${ADDED_DIR}/jboss-settings.xml $HOME/.m2/settings.xml
 
+# Add s2i scripts
+chmod g+x $ADDED_DIR/s2i/*
+mkdir -p /usr/local/s2i
+cp -p ${ADDED_DIR}/s2i/* /usr/local/s2i/
```

Comment:
I would prefer to set permission here, instead of setting them in $ADDED_DIR and then copy with preserve - its harder to read. But its very minor.